src/auth-service/config/firebase-admin.js

```diff
@@ -2,112 +2,114 @@ const admin = require("firebase-admin");
 const constants = require("./constants");
 const { initializeApp } = require("firebase-admin/app");
 const serviceAccount = require(`${constants.GOOGLE_APPLICATION_CREDENTIALS}`);
-const functions = require("firebase-functions");
-const createUserUtil = require("../utils/create-user");
-const { logObject } = require("../utils/log");
-const { default: isEmail } = require("validator/lib/isEmail");
-const isEmpty = require("is-empty");
-
-initializeApp({
-  credential: admin.credential.cert(serviceAccount),
-  databaseURL: constants.FIREBASE_DATABASE_URL,
-});
+// const functions = require("firebase-functions");
+// const createUserUtil = require("../utils/create-user");
+const { logObject, logText } = require("@utils/log");
+// const isEmpty = require("is-empty");
+// const emailMessages = require("../utils/email.msgs");
 
 const log4js = require("log4js");
 const logger = log4js.getLogger(
   `${constants.ENVIRONMENT} -- firebase-admin-config`
 );
 
-exports.newUserSignUp = functions.auth.user().onCreate(async (user) => {
-  try {
-    let email = user.email;
-    const firstName = user.firstName;
-    const message = `Welcome to AirQo, ${firstName}`;
-    const subject = "Welcome to AirQo!";
-
-    if (!isEmpty(email)) {
-      return await createUserUtil.sendFeedback({
-        email,
-        message,
-        subject,
-      });
-    }
-  } catch (error) {
-    logObject("error", error);
-    logger.error(`internal server error -- ${error.message}`);
-  }
+initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: constants.FIREBASE_DATABASE_URL,
 });
 
-exports.userDeleted = functions.auth.user().onDelete(async (user) => {
-  try {
-    const email = user.email;
-    const firstName = user.firstName;
-    const message = `AirQo Account Deleted, ${firstName}`;
-    const subject = "AirQo Account Successfully Deleted!";
+// exports.onUserCreate = functions.auth.user().onCreate(async (user) => {
+//   try {
```

Comment:
Just commented out all the logic for consuming directly from Firebase functions because it does not work right now. @BenjaminSsempala